_posts/developers/apps/tools/2019-01-01-00_overview.md

```diff
@@ -91,6 +91,10 @@ Beginning developers may want to start with one of the interface libraries which
 ### Rust
 
 * [solid-warp](https://gitlab.com/agentydragon/solid-warp) - simple example of authenticated reads/writes to a Solid pod
+
+### Spring + Kotlin
```

Comment:
There's no "Flask + Python" category either - people familiar with a language and its ecosystem will know whether use of a framework is relevant to them.

```suggestion
### Kotlin
```